README.md

```diff
@@ -1,36 +1,172 @@
-This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
 
-## Getting Started
+# Equilibrix: Smart Delta-Neutral Yield on DeFi
```

Comment:
Don't forget to mention that this repository is only for the frontend of this project